src/util/colors.js

```diff
@@ -54,3 +55,11 @@ export const defaultColorScale = getColorPalette(
     defaultColorScaleName,
     defaultClasses
 );
+
+// Correct colors not adhering to the css standard (add missing #)
+export const cssColor = color => {
+    if (!isString(color)) {
+        return color;
+    }
+    return (/(^[0-9A-F]{6}$)|(^[0-9A-F]{3}$)/i.test(color) ? '#' : '') + color;
```

Comment:
This could still be an invalid color (like "blorange") but I don't think we want to do a full CSS spec color comparison so that's fine :-)

Reply:
Yes, that might happen - but these don't exist in the system at this point :-)

Reply:
👍